swagger_templates/python/api_client.mustache

```diff
@@ -74,6 +82,11 @@ class ApiClient(object):
         self.cookie = cookie
         # Set default User-Agent.
         self.user_agent = '{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{packageVersion}}}/python{{/httpUserAgent}}'
+        # This is used for detecting for the special case of a path parameter
```

Comment:
This code was added to help with detecting the "__x-isi-url-encode-path-param__". This is slightly different than before. It used to do this inside the path parameter loop, but this is more efficient.